server/src/main/kotlin/com/github/neho4y/follower/domain/FeedbackFollower.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.neho4y.follower.domain
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "feedback_follower")
+data class FeedbackFollower(
+    @Column(name = "feedback_id")
+    val feedbackId: Long,
+
+    @Column(name = "user_id")
+    val userId: Long,
+
+    @Column(name = "follower_type")
+    @Enumerated(EnumType.STRING)
+    val followerType: FeedbackFollowerType,
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    val id: Long = 0
+)
+
+enum class FeedbackFollowerType {
+    ASSIGNEE,
+    WATCHER
```

Comment:
u can add followerType : AUTHOR

Reply:
Also consider single author validation

Reply:
We already have author field in the Feedback object. Followers supposed to be used for notifications. And author will be a watcher, so one may unsubscribe from own feedback.